src/main/java/ee/tuleva/onboarding/payment/PaymentProviderConfiguration.java

```diff
@@ -16,19 +16,20 @@
 public class PaymentProviderConfiguration {
 
   @Getter private final Map<Bank, PaymentProviderBank> banks;
-  private Map<String, PaymentProviderBank> banksByBic;
+  private Map<String, PaymentProviderBank> banksByAccessKey;
 
   public PaymentProviderBank getPaymentProviderBank(Bank bank) {
     return banks.get(bank);
   }
 
-  public PaymentProviderBank getPaymentProviderBank(String bic) {
-    return banksByBic.get(bic);
+  public PaymentProviderBank getPaymentProviderBank(String accessKey) {
+    return banksByAccessKey.get(accessKey);
   }
 
   @PostConstruct
   private void mapByBic() {
```

Comment:
`mapByAccessKey` would be a proper name

Reply:
I disagree, map is a type. No point in referring a type in the variable

Reply:
Oh, I get it